services/portal/home/homeTasks.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
     }
     // wait 2 sec to let the page fully loaded
     I.wait(2);
-    I.captureBrowserLog();
+    // I.captureBrowserLog();
```

Comment:
This log is useful for debugging. Should we really disable it?

Reply:
This step was failing for HEAL PRs and running locally gave the output something like "No debugger, cannot return variables". I will enable it in a separate PR and do the exercise again with the dummy HEAL PR